### ISSUE TITLE
Connection lines for named pipes

### ISSUE THREAD

**Issue by MincingWords** [catherinedevlin/ipython-sql#102]
Hi - 

I'm unclear on (and am not seeing examples of) how the connection line should look for connecting to a MySQL instance via named pipe. I guess it would look something like

%sql np://root@MYSQL/thisdb
 or 
%sql pipe://root@MYSQL/thisdb

where "thisdb" is the database name. Can someone tell me definitively what the connect line is supposed to be in such an instance?

**Comment by ZelphirKaltstahl** [comment]
@MincingWords Can you point to a part of the documentation, which mentions support of the `np` or `pipe` protocol? It might not be supported, if it is not mentioned anywhere.

**Comment by MincingWords** [comment]
No, I really can't point to any such in the docs. I found some references elsewhere to the use of named pipe URLs for DB connections but if iPython-sql doesn't even support such then finding the exact incantation is rather moot.
